neuralprophet/df_utils.py

```diff
@@ -513,6 +511,42 @@ def check_single_dataframe(df, check_y, covariates, regressors, events, seasonal
     return df
 
 
+def create_dummy_datestamps(
```

Comment:
Looks good overall!

Let us separate the disabling (editing of model config) from the  timestamps creation.

So, for this function, let us not pass the model, but only the precise args that are needed.